torch/ao/quantization/fx/_decomposed.py

```diff
@@ -32,6 +32,20 @@ def _quant_min_max_bounds_check(quant_min, quant_max, dtype):
 
 @impl(quantized_decomposed_lib, "quantize_per_tensor", "CompositeExplicitAutograd")
 def quantize_per_tensor(input, scale, zero_point, quant_min, quant_max, dtype):
+    """ Affine quantization for the Tensor using the same quantization parameters to map
```

Comment:
can we add types, both annotated in the code and also in the docstring?  For example, it's not clear if `scale` is a scalar or a tensor.